Jellyfin.Server/Migrations.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections.Generic;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Model.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Server
+{
+    /// <summary>
+    /// The class that knows how migrate between different Jellyfin versions.
+    /// </summary>
+    internal static class Migrations
+    {
+        private static readonly IUpdater[] _migrations =
+        {
+            new Pre10_5()
```

Comment:
I think it would be better to name the migrations to reflect the action they are performing. For example, this could be SetEnableThrottlingToFalse. And the logging migration could be a separate class with it's own specific name.

Reply:
I agree, this isn't very clear. Luckily we have a bit more time to make small changes like this.

Reply:
With EF migrations, the file/class names also use a timestamp as a prefix i.e. `20200305_SetEnableThrottlingToFalse` which allows them to be sorted easily by name. Maybe in this case we could use the version number instead of a timestamp?

Reply:
I still want to keep one migration per version change, but I'll split the changes into separate methods in the migration class.